test/mocha/30-event-api.js

```diff
@@ -754,6 +754,109 @@ describe('Event Storage API', () => {
     });
   });
 
+  describe('updateMany API', () => {
+    it('should update many events', async () => {
+      const count = 50;
+      let i = 0;
+      const events = [];
+      while(i++ < count) {
```

Comment:
i'd be more comfortable seeing a more standard for loop here.

Reply:
Okay.

Reply:
Addressed: https://github.com/digitalbazaar/bedrock-ledger-storage-mongodb/pull/57/commits/2d4bef9355f8f15bcc826c39a1960015cfa78a6f